pkg/plugins/platform/local.go

```diff
@@ -49,6 +49,14 @@ func (l *LocalApp) Init(basePath string) error {
 		log.Fatalln(err)
 	}
 
+	web, err := l.FindContainerByType("web")
+	if err == nil {
+		containerApproot := web.Labels["com.ddev.approot"]
```

Comment:
Curious if we should handle the unset/empty value ("") here. What should the behavior be? should all commands work? none?